plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroConfluentContentSchemaReader.java

```diff
@@ -47,7 +51,11 @@ public AvroConfluentContentSchemaReader(SchemaRegistryClient schemaRegistryClien
         }
         checkState(dataSchemaLocation.isEmpty(), "Unexpected parameter: dataSchemaLocation");
         try {
-            return Optional.of(schemaRegistryClient.getLatestSchemaMetadata(subject.get()).getSchema());
+            SchemaMetadata schemaMetadata = schemaRegistryClient.getLatestSchemaMetadata(subject.get());
+            ParsedSchema schema = schemaRegistryClient.getSchemaById(schemaMetadata.getId());
+            checkArgument(schema instanceof AvroSchema, "schema should be an instance of AvroSchema");
+            return Optional.of(schema.rawSchema())
```

Comment:
Can `AvroSchema#rawSchema()` returns null? Haven't found anything indicating that it can't.

Reply:
Ideally it should not return null. Will add `ofNullable` for defensive check